server/rematch/settings/docker.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
docker settings should import production settings and modify/add as necessary, instead of declaring everything